docs/examples/google-core-iot/README.md

```diff
@@ -141,3 +141,36 @@ Received message 'b'Hello world'' on topic '/devices/x.d.wott.local/commands' wi
 ## Reference implementation
 
 You may also want to take a look at our Balena [reference implementation](https://github.com/WoTTsecurity/wott-agent-balena/tree/master/google-core-iot) of the above.
+
+## Automatic enrolling/updating script 
+
+If you have a fleet of devices, it is difficult to perform the above steps from time to time to renew the certificates or upload new ones to just created devices.
+It would be right to have a script that allows you to perform all these tasks, as well as register new devices in the Google IoT registry, according to the list of devices in WoTT Dash.
+The following example script allows you to do this. This works in the following scenario.
+
+- Retrieve all devices from WoTT Dash
+- Retrieve all devices from Google IoT Registry.
+- Check all" Google devices " for updated certificates in the WoTT list. And update if so.
```

Comment:
Sounds weird

Reply:
Changed that part a bit.

Reply:
Don't worry about wording, I can touch that up later.